nx.json

```diff
@@ -40,6 +34,7 @@
     "@nx/jest:jest": {
       "inputs": [
         "default",
+        "^production",
```

Comment:
This was important - we want the test tasks of projects to be affected when we make changes to the production files of dependencies (which is what the `^` denotes,  "of dependencies")